### ISSUE TITLE
EnglishInflector incorrectly pluralizes "coupon"

### ISSUE THREAD

**Issue by lexmes** [symfony/symfony#40467]
**Symfony version(s) affected**: 5.1.2

**Description**
The `Symfony\Component\String\Inflector\EnglishInflector` incorrectly pluralizes the string "coupon" to "coupa". English is not my first language and i've never heard the word "coupa" before. I've searched for the plural version of coupon and didn't find "coupa" anywhere.

**How to reproduce**
With these two lines the problem should be reproducable:

```php
$inflector = new EnglishInflector();
$plural = $inflector->pluralize( "coupon" )[ 0 ];
echo $plural; // echoes "coupa"
```

**Additional context**
symfony/string 5.1.2

**Comment by Nyholm** [comment]
Thank you @lexmes.

This bug has been patched in 4.4 and will be merged up to 5.2 and 5.x to be released in the next patch version. Note that 5.1 is EOL and will not be updated.

**Comment by lexmes** [comment]
Thank you @Nyholm for the quick fix! :+1:
